documentation/src/test/java/example/ParameterizedTestDemo.java

```diff
@@ -109,36 +110,41 @@ void nullEmptyAndBlankStrings(String text) {
 
 	// tag::EnumSource_example[]
 	@ParameterizedTest
-	@EnumSource(TimeUnit.class)
-	void testWithEnumSource(TimeUnit timeUnit) {
-		assertNotNull(timeUnit);
+	@EnumSource(ChronoUnit.class)
```

Comment:
Although `ChronoUnit implements TemporalUnit`, that's a rather special case, and I'm not sure it benefits the average user to demonstrate that.

Reply:
My intent was to demonstrate under which circumstances one needs to specify the enum class explicitly. Thus the switch from `TimeUnit` to `ChronoUnit`. I'm open to suggestions, of course. 🙂

Reply:
Shall I add a clarifying sentence?

Reply:
Yes, please do.

Reply:
Done in cfae38b44235ec077444d36cf944c7e26b6961e5.